site/_includes/macros/cards/blog-card.njk

```diff
@@ -31,7 +31,9 @@
   </h2>
 
   <p class="type--caption gap-top-200">
```

Comment:
Would you mind updating the PR to look like this:

```njk
{% if not pd.hero %}
  <p class="type--caption gap-top-200">
    {{ pd.description }}
  </p>
{% endif %}
```

Otherwise it will create an empty paragraph with extra margin on the top

Reply:
my bad, will commit the changes.